.github/actions/load-image/action.yml

```diff
@@ -0,0 +1,42 @@
+name: 'load-image'
+description: "Tries to load image specified by tag from cache by deriving cache key from image name. Fails by default on cache miss."
+
+inputs:
+  docker-img:
+    description: 'Source image tag'
+    required: true
+  exit-on-cache-miss:
+    description: 'Determines whether job will exit with failure on cache-miss'
+    default: true
```

Comment:
Should not the default be `false`? It seems to be the main use-case

Reply:
I put `true` because that's the desired behaviour at 15 places using `load-image`, whereas the only place we want `false` is when we call this from `build-image` on attempt to load up older docker images to optimize build.